packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -216,4 +219,80 @@ describe('HCP home page', () => {
+    const cityInput = within(dialogTeam).getByRole('textbox', { name: 'City (State / Province)' })
+    const phoneNumberInput = within(dialogTeam).getByRole('textbox', { name: 'Phone number' })
+    const prefixPhoneNumber = within(dialogTeam).getByText('+33')
+    const emailInput = within(dialogTeam).getByRole('textbox', { name: 'Email' })
+
+    // Team creation button disabled and all fields empty
+    expect(nameInput).toHaveTextContent('')
+    expect(address1Input).toHaveTextContent('')
+    expect(address2Input).toHaveTextContent('')
+    expect(zipcodeInput).toHaveTextContent('')
+    expect(cityInput).toHaveTextContent('')
+    expect(phoneNumberInput).toHaveTextContent('')
+    expect(emailInput).toHaveTextContent('')
+    expect(dialogTeam).toBeInTheDocument()
+    expect(prefixPhoneNumber).toHaveTextContent(PhonePrefixCode.FR)
+    expect(createTeamButton).toBeDisabled()
+
+    // Team dropdown with prefix phone number
+    fireEvent.mouseDown(within(screen.getByTestId('team-edit-dialog-select-country')).getByRole('button'))
+    fireEvent.click(screen.getByRole('option', { name: 'Austria' }))
+    expect(prefixPhoneNumber).toHaveTextContent(PhonePrefixCode.AT)
+
+    // Team creation button activated and all fields filled
+    await userEvent.type(nameInput, lastName)
+    await userEvent.type(address1Input, '5 rue')
+    await userEvent.type(cityInput, 'Grenoble')
+    await userEvent.type(phoneNumberInput, '0600000000')
+    await userEvent.type(zipcodeInput, '38000')
+    await userEvent.type(emailInput, 'toto@titi.com')
+    expect(createTeamButton).not.toBeDisabled()
+
+    // Team creation button disabled and a field in error with the error message
+    userEvent.clear(zipcodeInput)
+    expect(createTeamButton).toBeDisabled()
+    await userEvent.type(zipcodeInput, '75d')
+    expect(within(dialogTeam).getByText('Please enter a valid zipcode')).toBeVisible()
+    expect(createTeamButton).toBeDisabled()
+    userEvent.clear(zipcodeInput)
+    await userEvent.type(zipcodeInput, '75800')
+    expect(createTeamButton).not.toBeDisabled()
+
+    userEvent.clear(phoneNumberInput)
+    expect(createTeamButton).toBeDisabled()
+    await userEvent.type(phoneNumberInput, '060')
+    expect(within(dialogTeam).getByText('Please enter a valid phone number')).toBeVisible()
+    expect(createTeamButton).toBeDisabled()
+    userEvent.clear(phoneNumberInput)
+    await userEvent.type(phoneNumberInput, '06000000')
+    expect(createTeamButton).not.toBeDisabled()
+
+    userEvent.clear(emailInput)
+    await userEvent.type(emailInput, 'tototiti.com')
+    expect(within(dialogTeam).getByText('Invalid email address (special characters are not allowed).')).toBeVisible()
+    expect(createTeamButton).toBeDisabled()
+    userEvent.clear(emailInput)
+    await userEvent.type(emailInput, 'toto@titi.com')
+    expect(createTeamButton).not.toBeDisabled()
+
+    userEvent.click(createTeamButton)
```

Comment:
```suggestion
    await act(async () => {
      userEvent.click(createTeamButton)
    })
```
Without this. You have error in the console. I suggest you try it locally to see (with and without). If you don't understand this fix I can explain it to you

Reply:
Without your correction I have the tests on the glass and when I add 
await act(async () => {
      userEvent.click(createTeamButton)
    }) the tests are red

Reply:
Are you sure? It is working on my side